api/src/wfl/once.clj

```diff
@@ -52,3 +52,24 @@
     (if (util/getenv "WFL_DEPLOY_ENVIRONMENT")
       (service-account-token)
       (util/shell! "gcloud" "auth" "print-access-token"))))
+
+(defn- read-oauth-client-id
+  "Actively read the OAuth client ID from vault or the environment"
```

Comment:
As opposed to passively read?

Reply:
Just wanted to make it clear that this actually read stuff, guess it is moot now

Reply:
Haha!